web/api/v1/api.go

```diff
@@ -182,6 +182,11 @@ func (api *API) queryRange(r *http.Request) (interface{}, *apiError) {
 	if err != nil {
 		return nil, &apiError{errorBadData, err}
 	}
+	if end <= start {
+		err := errors.New("start timestamp must be before end timestamp. Try switching them around")
```

Comment:
Both timestamp being equal can be a legal and meaningful query.

The error message itself should not contain periods. I'd just leave out the 2nd half.